apiClient.go

```diff
@@ -19,13 +19,20 @@ import (
 	"io/ioutil"
 	"net/http"
 	"time"
-
-	"github.com/spf13/viper"
 )
 
-// A Client holds the packages *viper.Viper and *http.Client. To get a *Client, call New.
+// A LWAPIConfig holds the
```

Comment:
did you forget to finish typing the rest of this comment after `the`?

Reply:
.... yup, sure looks that way.